content/english/anti-vegan/humans-are-superior.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Humans Are Superior"
+date: 2019-10-12T07:28:15+13:00
```

Comment:
Can you add this in after the `date` please to populate the contributors?
```
contributors:
    - 
        name: sam-martin
        role: editor
    - 
        name: tery-hardwicke
        role: author
```
(We need to add Val at, but I'm just waiting for her bio :) )